pkg/roachprod/vm/vm.go

```diff
@@ -511,7 +511,7 @@ type Provider interface {
 	// GetHostErrorVMs returns a list of VMs that had host error since the time specified.
 	GetHostErrorVMs(l *logger.Logger, vms List, since time.Time) ([]string, error)
 	// GetVMSpecs returns a json list of VM specs, according to a specific cloud provider.
```

Comment:
Nit: comment is imprecise. It should mention that the resulting map is from `VM.Name` to a map of VM attributes, which are specific per cloud provider.

Reply:
Thanks for the catch!